src/hmrc-design-patterns/unique-taxpayer-reference/index.njk

```diff
@@ -1,5 +1,6 @@
 ---
 title: Unique Taxpayer Reference
+menuText: Unique Taxpayer Reference
```

Comment:
@BenSpears1 I'm not 100% sure on this but I don't think you need to put in menuText unless you want some custom wording. You might be able to leave this out?

Reply:
We're doing it to force the capitalisation @timsb